Neos.Flow/Configuration/Caches.yaml

```diff
@@ -93,16 +93,23 @@ Flow_Persistence_Doctrine_Results:
 Flow_Persistence_Doctrine_SecondLevel:
   backend: Neos\Cache\Backend\SimpleFileBackend
 
-# Flow_Reflection*
 #
+# DEPRECATED since 8.4
+# These caches will be removed in 9.0, they are unused so you can safely remove any configuration for them.
 #
 Flow_Reflection_Status:
   frontend: Neos\Cache\Frontend\StringFrontend
   backend: Neos\Cache\Backend\SimpleFileBackend
 Flow_Reflection_CompiletimeData:
   backend: Neos\Cache\Backend\SimpleFileBackend
-Flow_Reflection_RuntimeData: []
-Flow_Reflection_RuntimeClassSchemata: []
+
+# Flow_Reflection
+#
+#
+Flow_Reflection_RuntimeData:
+  backend: Neos\Cache\Backend\SimpleFileBackend
+Flow_Reflection_RuntimeClassSchemata:
+  backend: Neos\Cache\Backend\SimpleFileBackend
```

Comment:
Why do these get an explicit backend configured now?

Reply:
Because they don't use any features the (more complex) FileBackend offers, and can therefore just use the SimpleFileBackend (should have been the case since ever).